WcaOnRails/app/views/regional_organizations/list.html.erb

```diff
@@ -0,0 +1,23 @@
+<% provide(:title, 'Regional Organizations list') %>
+
+<div class="container">
+  <h1><%= yield(:title) %></h1>
+
+  <% if @regional_organizations.empty? %>
+    <%= alert :warning, "There are currently no Regional Organizations." %>
+  <% else %>
+    <% unless @regional_organizations.where(currently_acknowledged: true).empty? %>
+      <h2>Currently acknowledged:</h2>
+      <%= render "regional_organizations/regional_organizations_table", regional_organizations: @regional_organizations.where(currently_acknowledged: true) %>
+    <% end %>
+
+    <% unless @regional_organizations.where(currently_acknowledged: false).empty? %>
```

Comment:
My intention isn't for `currently_acknowledged == false` to mean "never acknowledged". This is just to have a list somewhere of RO's that aren't currently acknowledged in case it's needed, for instance to re-use an already existing entry in the DB of a RO that was previously recognized but it isn't anymore instead of creating a new one. Does that make sense?